lib/bulkrax.rb

```diff
@@ -104,9 +108,8 @@ class << self
         "resource_type" => { from: ["type"], parsed: true },
         "remote_files" => { from: ["thumbnail_url"], parsed: true }
       },
-      "Bulkrax::CsvParser" => {
-        "remote_files" => { from: ["remote_files"], parsed: true }
-      },
+      # When empty, a default_field_mapping will be generated
+      "Bulkrax::CsvParser" => {},
```

Comment:
I'm unsure why you added remote_files here @orangewolf ... but I don't see why it's needed? If the CSV contains a remote_files column, it will be imported. I suspect this could be specific to Atla?

Reply:
I added that because this code needs to always be called for remote files columns: `https://github.com/samvera-labs/bulkrax/blob/master/app/matchers/bulkrax/application_matcher.rb#L50` 

Leaving it in the matcher means you can still override it, but if you don't wrap the remote files in `{ url: #{value}}` then the files won't actually import. 

We could fix the spec (maybe set them to blank in the tests) or force remote_file to be parsed all the time (which I think is less explicit and harder to find).  You likely want it on by default for bagit and xml parser too.

Reply:
Actually I was thinking about checking whether a parse method exists and setting parse to true within the lambda that generates the default mapping ... I think that would fix this in a cleaner way?

Reply:
yep, that would work